src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -62,6 +65,10 @@
 
     @State
     @Nullable
+    ArrayList<Trigger> triggers;
```

Comment:
I know and follow that rule every time, but Icepick — the library which handles state on configuration changes — doesn’t accept generics, because in that case it doesn’t know how to put the field to a `Parcel` container — it doesn’t support generics as well.